### DISCUSSION TITLE
Replace 0auth eith supertokens?

### DISCUSSION THREAD

**Discussion by Jarrodsz** [sergiodxa/remix-auth-oauth2#38] [Q&A] [answered]
Is it possible to replace 0auth with supertokens? Its opensource alternative 0auth

**Comment by sergiodxa** [comment, marked as answer]
Supertokens needs you to use a frontend SDK, Remix Auth strategies are 100% server-side, also Supertokens uses OAuth for social logins, but they don't support being used as a OAuth provider which means you need to use their frontend SDK
